src/args.rs

```diff
@@ -18,7 +18,24 @@ pub enum Cli {
 #[derive(Debug, StructOpt)]
 pub struct Args {
     /// List defined favorite templates from the config
-    #[structopt(long)]
+    #[structopt(
+        long,
+        conflicts_with = "git",
+        conflicts_with = "subfolder",
+        conflicts_with = "path",
+        conflicts_with = "branch",
+        conflicts_with = "name",
+        conflicts_with = "force",
+        conflicts_with = "template_values_file",
+        conflicts_with = "silent",
+        conflicts_with = "config",
```

Comment:
issue: `config` should not be conflicting since it relates to the favourites files